Index: main.py
===================================================================
diff --git a/main.py b/main.py
deleted file mode 100644
--- a/main.py	(revision 8b1b6be8ff17f55b1551978a6282494429d3cdac)
+++ /dev/null	(revision 8b1b6be8ff17f55b1551978a6282494429d3cdac)
@@ -1,5 +0,0 @@
-import eel
-
-eel.init("web")
-
-eel.start("main.html", size=(700, 7))
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 8b1b6be8ff17f55b1551978a6282494429d3cdac)
+++ /dev/null	(revision 8b1b6be8ff17f55b1551978a6282494429d3cdac)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: web/main.html
===================================================================
diff --git a/web/main.html b/web/main.html
deleted file mode 100644
--- a/web/main.html	(revision 8b1b6be8ff17f55b1551978a6282494429d3cdac)
+++ /dev/null	(revision 8b1b6be8ff17f55b1551978a6282494429d3cdac)
@@ -1,15 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <meta charset="UTF-8">
-    <title>Кодеинчик</title>
-
-    <script src="eel.js"></script>
-    <link rel="icon" type="image/png" href="/icon.png"
-</head>
-<body>
-
-    <h1>Кодеин</h1>
-
-</body>
-</html>
